serverless/event_sources/serverless-kafka-source.adoc

```diff
@@ -6,7 +6,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The Apache Kafka event source brings messages into Knative. It reads events from an Apache Kafka cluster and passes these events to an event sink so that they can be consumed. You can use the `KafkaSource` event source with {ServerlessProductName}.
+The Knative Kafka event source reads events from an Apache Kafka cluster and passes these events to a sink so that they can be consumed.
```

Comment:
I can't remember if we've talked about this before, but:
>! consume _verb_
Do not use when a simpler term such as "use" is possible. "Consume" is acceptable when it is part of the existing product terminology.

We were previously following the guidance to only use consume when it meant that the original asset was destroyed on use.

Reply:
I can probably just leave it at "The Knative Kafka event source reads events from an Apache Kafka cluster and passes these events to a sink."?